airflow/cli/commands/variable_command.py

```diff
@@ -66,44 +68,82 @@ def variables_delete(args):
     Variable.delete(args.key)
 
 
+DIS_RESTRICT = 'restrict'
+DIS_OVERWRITE = 'overwrite'
+DIS_IGNORE = 'ignore'
+CREATED = 'created'
+DISPOSITIONS = [DIS_RESTRICT, DIS_OVERWRITE, DIS_IGNORE]
+
+
+def _prep_import_status_msgs(var_status_map):
```

Comment:
If possible, it is better to avoid abbreviations in function names.

Reply:
Done - updated function name.